requirements/edx/kernel.in

```diff
@@ -116,6 +116,9 @@ openedx-django-require
 # openedx-events 3.1.0 introduces producer API
 openedx-events>=3.1.0               # Open edX Events from Hooks Extension Framework (OEP-50)
 openedx-filters                     # Open edX Filters from Hooks Extension Framework (OEP-50)
+# FIXME Use pip package once created
+# ref https://github.com/openedx/openedx-learning/pull/62
+git+https://github.com/open-craft/openedx-learning.git@jill/smarter-object-tags-rebase#egg=openedx-learning==0.1
```

Comment:
To do -- update this dependency when PR merged, tagged, and released on pypi

Reply:
Maybe a dummy merge PR openedx-learning to try to force it to push to PyPI?

Reply:
I worked it out at last.. needed to push a new tag, and that triggered the release :)

https://pypi.org/project/openedx-learning/